### DISCUSSION TITLE
.env for Local Setup

### DISCUSSION THREAD

**Discussion by mohit-override** [mfts/papermark#815] [Q&A] [answered]
How to populate variables in .env file ?
Its not mentioned in README

![image](https://github.com/user-attachments/assets/df08caa6-de51-46be-a739-a833edb7534a)

**Comment by mfts** [comment, marked as answer]
@mohit-gogitter 

You need to set up a postgres instance. Either hosted or local or via docker. That'll give you the connection string. 

You can either set up a blob storage on aws s3, vercel or minio (via docker) to get the file storage env variables. 

For emails you can use resend to set up an account there but it also works without emails

**Reply by mohit-override** [reply]
got it, thanks

**Reply by heymaaz** [reply]
I have set up postgres and minio via docker. But when I run `npm run dev` I get it running at localhost:3000. When I use a dummy email to sign in I get the following error and can't get past the login screen/
<img width="1624" alt="Screenshot 2024-11-06 at 7 40 55 PM" src="https://github.com/user-attachments/assets/56b6f1d0-0f74-4895-ade0-6ca270f71ed1">
